### PR TITLE
Kube-hunter Parser

### PR DESCRIPTION
**Pull request by a-ruff**
**Description**

Adding a new parser for the kubernetes scanner : https://github.com/aquasecurity/kube-hunter

**Test results**

Basics parsers tests.

### CODE REVIEW

**Review by manuel-sommer — CHANGES_REQUESTED** [review]
Could you please also add the new parser to the documentation:
https://github.com/DefectDojo/django-DefectDojo/tree/master/docs/content/en/integrations/parsers/file

File: dojo/tools/kubehunter/parser.py [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+            # Finding mitigation and reference
+            avd_reference = item.get('avd_reference')
+
+            if avd_reference and avd_reference != '' and vulnerability_id != 'None':
+                mitigation = f"Further details can be found in kube-hunter documentation available at : {avd_reference}"
+                references = "**Kube-hunter AVD reference**: " + avd_reference
+            else:
+                mitigation = ''
+                references = ''
```
Comment (kiblik) [inline dojo/tools/kubehunter/parser.py:55]: If I remember correctly, empty `mitigation` and `references` should be `None`, not `''`

**Comment by a-ruff** [comment]
Thanks for your feedback, I've made the appropriate changes.

**Comment by mtesauro** [comment]
@a-ruff Just merged Celery 5.3.4 which reverts breaking changes in 5.3.2 and 5.3.3

Can you please rebase? Thanks!

**Comment by a-ruff** [comment]
I rebased on dev branch by following [DefectDojo documentation](https://raw.githubusercontent.com/DefectDojo/django-DefectDojo/3ef065728d85358f254b56375179494861d8c3be/.github/pull_request_template.md) . Please let me know if I need to do anything else.

**Review by Maffooch — APPROVED** [review]

**Review by mtesauro — APPROVED** [review]
Approved

**Review by grendel513 — APPROVED** [review]

**Review by cneill — APPROVED** [review]

**Pull request merged**